monitoring/README.md

```diff
@@ -66,13 +66,40 @@ repository is created in a directory named `viya4-monitoring-kubernetes-<release
 
 ### <a name="mon_custom"></a>Customize the Deployment
 
-The process of customizing the monitoring deployment consists of: 
+For most deployment scenarios, the process of customizing the monitoring deployments consists of: 
 - creating the location for your local customization files
 - using the `USER_DIR` environment variable to specify the location of the customization files
 - copying the customization files from one of the provided samples to your local directory
 - specifying customization variables and parameters in the customization files
 
-See the [main README](../README.md#customization) to for information about the customization process.
+Other scenarios use different customization steps that are specific to each scenario.
+
+[Samples](#msample) are provided for several common deployment scenarios. Each sample includes detailed information about the customization process and values for the scenario.
+
+#### Creating the Location for Customization Files
+
+The `USER_DIR` environment variable enables you to use a directory outside of the local repository to contain the customization files. By using a location outside of the repository, you can put the customization files in a location that is under version control, and you can preserve your customizations when you deploy new versions of the monitoring and logging applications. You could also create different sets of configuration files for different cluster types, and then use the `USER_DIR` variable to specify the configuration files to use based on the cluster type on which you were deploying.
+
+For example, to create a customization directory named *my-viya4mon-user-dir* that contains a directory for monitoring customization files:
+
+```bash
+mkdir -p ~/my-viya4mon-user-dir/monitoring
+```
```

Comment:
I wonder if it is confusing to readers to create this multi-level directory (i.e. all the way down to the monitoring sub-dir) and then, as directed in the next section, point the USER_DIR env var one level higher.  We could: 

-  break the make directory step into two steps (i.e. create my-viya4mon-user-dir and then create the monitoring sub-dir under it);
- or, maybe just point out that the env var is pointing to the parent directory?

Reply:
I've revised the instructions for creating the directory to include two steps - creating the root and then the sub-directory and indicated that the root directory is where USER_DIR will point

Reply:
Looks good.  You can remove the -p option in the mkdir commands.  It's a flag that says "make any parent/intermediate directories that don't already exist" and no longer needed since we're now creating each level manually.

Reply:
Great - thanks. Fixed as noted